OsiguSDK.Examples/ClaimsProviderExamples.cs

```diff
@@ -94,5 +94,11 @@ public Pagination<Claim> GetListOfClaims(int? page = 0, int? size = 25)
         {
             return _client.GetListOfClaims();
         }
+
+        public void VoidClaim(string claimdId)
+        {
```

Comment:
do we need this empty method?

Reply:
no we don't. @dmarenco please delete this empty method